pkg/ui/workspaces/db-console/BUILD.bazel

```diff
@@ -1,55 +1,169 @@
 load("@npm//webpack-cli:index.bzl", webpack = "webpack_cli")
 
-WEBPACK_DATA_OSS = [
-    ".babelrc",
-    ".eslintrc.json",
-    ".stylintrc",
-    "assets",
-    "favicon.ico",
-    "fonts",
-    "src",
-    "styl",
-    "tsconfig.json",
-    "webpack.app.js",
-    ":node_modules",
-    "//pkg/ui:node_modules_db_console",
-    "//pkg/ui/workspaces/cluster-ui:cluster_ui_package",
-    "//pkg/ui/workspaces/db-console/src/js:ui_protos_oss_package",
-    "@npm//@babel/parser",
-    "@npm//@babel/plugin-proposal-class-properties",
-    "@npm//@babel/plugin-proposal-object-rest-spread",
-    "@npm//@babel/plugin-transform-modules-commonjs",
-    "@npm//@babel/plugin-transform-runtime",
+# do not include following dependencies:
```

Comment:
Maybe worth adding a `TODO` to add some static checking or generation to this. Keeping it up-to-date manually seems like a pain. Incorporating it into `dev generate bazel` would be ideal but it might not be possible to do it efficiently enough.